src/arrayMethodJoin.js

```diff
@@ -4,9 +4,22 @@
  * Implement method join
  */
 function applyCustomJoin() {
-  [].__proto__.join2 = function(separator) {
-    // write code here
+  [].__proto__.join2 = function(separator = ',') {
+    let joinResult = '';
```

Comment:
what is join Result? join it's an action, it can't be the name for the variable (it can be name for some function)